x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/action/rolemapping/ReservedRoleMappingAction.java

```diff
@@ -34,6 +35,8 @@
  */
 public class ReservedRoleMappingAction implements ReservedClusterStateHandler<List<PutRoleMappingRequest>> {
     public static final String NAME = "role_mappings";
+    private static final String FILE_SETTINGS_METADATA_NAMESPACE = "file_settings";
```

Comment:
Consider moving these. These constants are not available in the security plugin so need to be defined somewhere.